pp/peers/start.go

```diff
@@ -31,8 +31,8 @@ func InitPeer(registerFn func()) {
 	go ListenOffline()
 }
 
-// RegisterChain
-func RegisterChain(toSP bool) {
+// RegisterToSP
+func RegisterToSP(toSP bool) {
```

Comment:
I feel like the new name doesn't make sense if we keep a toSP boolean as argument